docs/tools/firmware.sh

```diff
@@ -658,27 +937,23 @@ case "$os" in
 				echo -e "\nNote: In order to get complete firmware files, download macOS Monterey or later.\n"
 				python3 fetch-macOS-v2.py
 				echo -e "\nConverting image from .dmg to .img"
-				if [[ ${verbose} = -v ]]
+				if [[ ${verbose} = "-v" ]]
 				then
 					dmg2img -v BaseSystem.dmg fw.img
 				else
 					dmg2img -s BaseSystem.dmg fw.img
 				fi
+				rm BaseSystem.dmg
```

Comment:
rm ${verbose}

Reply:
Btw in cleanup_dmg function, workdir gets removed, which also has the dmg, so I wonder what's the actual advantage of specifically removing it here, since the next step before cleanup is simply firmware extraction and reloading drivers.

Reply:
tmpfs is usually in ram, if you are running low on ram the script crashes. The DMG is almost 2GB so this saves a lot of ram space.

Reply:
> tmpfs is usually in ram, if you are running low on ram the script crashes. The DMG is almost 2GB so this saves a lot of ram space.

But it is not removed unless img is made, so what's the benefit here?

Reply:
When I was testing the script it would crash a lot here and complain about low disk space, but once I added the `rm` it usually worked.

Reply:
Im going to move the dmg dir to somewhere on disk.